binaries/upload.py

```diff
@@ -23,21 +23,29 @@
 PACKAGES = ["torchserve", "model-archiver", "workflow-archiver"]
 
 
-def upload_pypi_packages():
+def upload_pypi_packages(test_pypi=False):
     """
     Takes a list of path values and uploads them to pypi using twine, using token stored in environment variable
     """
     os.system(f"pip3 install twine -q")
 
     # Note: TWINE_USERNAME and TWINE_PASSWORD are expected to be set in the environment
     for dist_path in [TS_WHEEL_PATH, MA_WHEEL_PATH, WA_WHEEL_PATH]:
-        exit_code = os.system(f"twine upload {dist_path}/* --repository-url https://test.pypi.org/legacy/ --verbose")
+        if test_pypi:
```

Comment:
The two commands are identical in `if` and `else` except for repo URL so we can collapse this down with a single `f""` string